tests/adapter/dbt/tests/adapter/utils/fixture_array_construct.py

```diff
@@ -0,0 +1,12 @@
+# array_construct
+
+models__array_construct_expected_sql = """
```

Comment:
not sure how other folks feel but the python standard for constants is to use upper case, [see](https://peps.python.org/pep-0008/#constants). As I understand the previous convention was based on emulating how we match macros in jinja but since we're not doing that here it's not necessary.

Reply:
The lowercase capitalization you see here is consistent with the tests I'm familiar with. You can see a sample of those existing tests via `git grep '_sql = """' ./`.

In terms of broader compliance with the PEP 8 style guide, we do use `flake8` checks in CI and in `pre-commit` checks. I haven't read the nitty gritty details of what `flake8` does/doesn't cover though. You can see our flake8 config [here](https://github.com/dbt-labs/dbt-core/blob/main/.flake8).